src/RetrievesHelper.ts

```diff
@@ -0,0 +1,31 @@
+import { R4 } from '@ahryman40k/ts-fhir-types';
```

Comment:
Even though this may change in the future, we should explicitly declare `@ahryman40k/ts-fhir-types` as a dependency of this repo too with `npm install --save` so it appears in package.json.

This only works because it's a dependency of fqm-execution. Since we're importing it directly, we should be explicit